README.md

```diff
@@ -127,7 +128,7 @@ https://www.pioneer2.net/community/threads/tethealla-server-setup-instructions.1
 in `patcher.go` that correspond to your client. If none exist, you'll have to find the offsets with a
 hex editor.
 
-### 7. Add files the patch directory
+### 7. Add files to the patch directory
```

Comment:
Man you can tell that I wrote most of these after coding all day, thanks for cleaning up these typos